qutebrowser/components/misccommands.py

```diff
@@ -223,41 +223,80 @@ def _insert_text_cb(elem: Optional[apitypes.WebElement]) -> None:
     tab.elements.find_focused(_insert_text_cb)
 
 
+def _wrap_find_at_pos(value: str, tab: apitypes.Tab,
+                      callback: Callable[[Optional[apitypes.WebElement]], None]
+                      ) -> None:
+    try:
+        x, y = map(int, value.split(',', maxsplit=1))
+    except ValueError:
+        message.error(f"'{value}' is not a valid point!")
```

Comment:
```suggestion
    except ValueError as e::
        message.error(f"'{value}' is not a valid point: {e}")
```

To make it clearer what the culprit is.